ci/test.sh

```diff
@@ -57,17 +63,27 @@ function prep_artifacts {
 function run_btests {
     banner "Running baseline tests: zeek"
 
+    if [[ -n "${ZEEK_CI_ZAM}" ]]; then
+        ALTERNATIVE_ARG="-a zam"
```

Comment:
Maybe `local alternative_arg` given it's just used in this function? And possibly initialize with `""`.